.github/workflows/tps-basic-test.yml

```diff
@@ -202,9 +202,19 @@ jobs:
           docker exec pki pki -n caadmin tps-token-show $CUID | tee output
           sed -n 's/\s*Status:\s\+\(\S\+\)\s*/\1/p' output > actual
           diff expected actual
-
+          
           docker exec pki pki -n caadmin tps-cert-find --token $CUID
 
+      - name: KRA key find
+        run: |
+          OWNER="UID=testuser"
+          echo "UID=testuser" > expected
+          docker exec pki pki -n caadmin kra-key-find --owner $OWNER | tee output
+          sed -n 's/\s*Owner:\s\+\(\S\+\)\s*/\1/p' output > actual
+          diff expected actual
+          
+          docker exec pki pki -n caadmin kra-key-find
```

Comment:
Thanks for the updates!

I think for keys generated by TPS the owners will follow this format: `<token>:<username>`, but for keys generated by CA or IPA the owners might follow different formats.

So for this test you'll need to get the token ID from an earlier TPS enrollment operation which was generated in line 168 and stored into a file called `cuid`, then concatenate that with the username specified in line 197.

If you want to see how the owners actually look like, you probably need to run the `pki kra-key-find` command with no filters first, then run the command with the filter afterwards.

Reply:
yeah, it was failing in CI tests. Running with kra-key-find then will pass owner option.

Reply:
It's working now. please check.